### DISCUSSION TITLE
PWAsForFirefox 2.2.0

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#251] [Announcements]
## Added

* Support for using SOCKS proxies (#218).
* Support for disabling HTTPS validation and custom certificates (#241).
* Autocomplete for most known web app categories (#180).

## Changed

* The link target is no longer changed when disabling "tabs mode" or "open out-of-scope URLs" if the other one is still enabled.
* It has been made more clear that Xcode Command Line Tools are required.
* Updated dependencies.

## Fixed

* Call the system integration when removing a profile (#243).
* Allow parsing manifests that include an empty color string (#236).

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.2.0'>PWAsForFirefox 2.2.0</a>.</em>

**Comment by TacoDelPaco** [comment]
Will Windows ARM64 support be coming anytime soon? 🥲

**Reply by filips123** [reply]
I think ARM should already work if you compile the program yourself from source. The main thing that prevents it from being distributed is that WiX Toolset does not support ARM fully. I will check if something has changed since the last time I checked it.

**Update:** Stable WiX 3.11 still does not support ARM. WiX 4.0 supports it, but it will have to wait, as it's still in the preview stage and `cargo-wix` does not support it yet. Hopefully, it will be released in a few months.
